packages/dendron-plugin-views/src/components/DendronConfigure.tsx

```diff
@@ -0,0 +1,175 @@
+  }, [engine.config]);
+
+  useConfig({ opts: workspace });
+  const { Header, Content, Sider } = Layout;
+  const items = Object.keys(dendronConfig).map((conf) => {
+    return {
+      key: conf,
+      label: _.startCase(dendronConfig[conf].group),
+    };
+  });
+  const menuItems = items.filter(
+    (value, index, self) =>
+      index === self.findIndex((t) => t.label === value.label)
+  );
+
+  const cleanTitle = (title: string) => {
+    return title
+      .split(".")
+      .map((k) => _.startCase(k))
+      .join(" > ");
+  };
+
+  const postMessage = ({ key, value }: { key: string; value: any }) => {
+    let newConf = _.cloneDeep(config);
+    newConf = newConf && _.set(newConf, key, value);
+    dendronConfig[key].default = value;
+    setConfig(newConf);
+    postVSCodeMessage({
+      type: ConfigureUIMessageEnum.onUpdateConfig,
+      data: { config: newConf },
+      source: DMessageSource.webClient,
+    } as ConfigureUIMessage);
+  };
+
+  const handleSearch = debounce((e: any) => {
+    setSearchString(e.target.value);
+  }, 500);
+
+  return (
+    <Layout style={{ height: "100vh", overflow: "hidden" }}>
+      {!config ? (
+        <Spin />
+      ) : (
+        <>
+          <Header className="header">
+            <Input placeholder="search config" onChange={handleSearch} />
+          </Header>
+          <Layout>
+            <Sider
+              width={160}
+              style={{
+                height: "100%",
+                borderRight: "1px solid #383838",
+                margin: 0,
+                padding: "20px 16px 0 21px",
+              }}
+            >
+              <Anchor
+                getContainer={() =>
+                  document.getElementById("configure-content")!
+                }
+              >
+                <Space
+                  direction="vertical"
+                  size="middle"
+                  style={{ display: "flex" }}
+                >
+                  {menuItems.map((item) => (
+                    <Anchor.Link
+                      key={item.key}
+                      href={`#${item.label.toLowerCase()}`}
+                      title={item.label}
+                    />
+                  ))}
+                </Space>
+              </Anchor>
+            </Sider>
+            <Layout>
+              <Content
+                id="configure-content"
+                style={{
+                  padding: 24,
+                  margin: 0,
+                  minHeight: 280,
+                  height: "100%",
+                  overflowY: "scroll",
+                }}
+              >
+                <Space
+                  direction="vertical"
+                  size="middle"
+                  style={{ display: "flex" }}
+                >
+                  {Object.keys(dendronConfig)
+                    .filter((conf) =>
+                      _.lowerCase(conf).includes(_.lowerCase(searchString))
+                    )
+                    .map((conf) => {
```

Comment:
could we do some manual rearranging of the order of the setting groups? 

I'm thinking maybe
1. Commands
2. Workspace
3. Preview
4. Publishing
5. Global
6. Dev

Current:
<img width="147" alt="image" src="https://user-images.githubusercontent.com/3203268/178399059-b69e12bf-5699-44b4-bca3-2aa7c4963526.png">